build.gradle

```diff
@@ -174,8 +174,11 @@ dependencies {
   implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.9'
   implementation group: 'commons-io', name: 'commons-io', version: '2.6'
   implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
+  implementation files("${System.properties['java.home']}/../lib/tools.jar")
   implementation configurations.junit.dependencies
   implementation files(project(':replacecall').sourceSets.main.output)
+  def cfLocalHome = String.valueOf(System.getenv("CHECKERFRAMEWORK"))
```

Comment:
Let's not make Randoop depend on the Checker Framework.  See comment below about the Scene Library vs BCEL.